pkgs/development/interpreters/php/default.nix

```diff
@@ -133,8 +135,8 @@ let
       ++ optional mhashSupport "--with-mhash"
       ++ optional curlSupport "--with-curl=${curl.dev}"
       ++ optional zlibSupport "--with-zlib=${zlib.dev}"
-      ++ optional libxml2Support "--with-libxml-dir=${libxml2.dev}"
-      ++ optional (!libxml2Support) [
+      ++ optional (libxml2Support && (versionOlder version "7.4")) "--with-libxml-dir=${libxml2.dev}"
```

Comment:
What happens to libxml support on 7.4?

Reply:
Fixed, for php 7.4 the parameter "--with-libxml-dir = $ {libxml2.dev}" is not used